design/open/evict-user-from-auth-cache-on-reset-password.md

```diff
@@ -0,0 +1,13 @@
+## The problem: 
+When a user logs in to lakeFS we save his credentials in the auth-cache. 
```

Comment:
```suggestion
When a user logs in to lakeFS we save their credentials in the auth-cache. 
```
and also below.

See [How to write great documentation for lakeFS, "style basics"](https://docs.google.com/document/d/1vif05nDiKZYPwYdINLoq4BJ6j7bBIQ6ZA2dt7McuzyY/edit#heading=h.dqp77semtfge).